products/joomla.md

```diff
@@ -17,21 +17,21 @@ auto:
 # eol see https://developer.joomla.org/roadmap.html
 releases:
 -   releaseCycle: "5"
-    releaseDate: 2023-10-14
+    releaseDate: 2023-10-17
     support: true
     eol: 2027-10-19
     latest: "5.0.0"
-    latestReleaseDate: 2023-10-14
+    latestReleaseDate: 2023-10-17
     # check later for removal: link with changelogTemplate does not work (yet)
     link: 
       https://www.joomla.org/announcements/release-news/5900-joomla-5-0-and-joomla-4-4-are-here
 
 -   releaseCycle: "4"
     releaseDate: 2021-08-17
-    support: 2023-10-17
+    support: 2024-10-15
```

Comment:
```suggestion
    support: 2024-10-17
```

According to https://developer.joomla.org/roadmap.html for 14.x (strange that 4.4 says otherwise though).